src/skewsymmetricmv.jl

```diff
@@ -26,48 +26,60 @@ using VectorizationBase, LinearAlgebra, Static
       Nr = N ÷ $CU
       @assert Nr * $CU == N# "TODO: handle remainders!!!"
       r = 0
-      @show Nr
-      for ro = 1:Nr
+      for ro = 1:1
         # down rows
         # initialize accumulators
         Base.Cartesian.@nexprs $RUI u -> v_u = VectorizationBase.vzero($W, $T)
         Base.Cartesian.@nexprs $CUI u -> s_u = VectorizationBase.vzero($W, $T)
         c = 0
         for co = 1:Nr
           # across columns
-          @show co
           Base.Cartesian.@nexprs $RUI ri -> begin
             rowind_ri = MM($W, r + (ri-1)*$W)
             sx_ri = VectorizationBase.vload(px, (rowind_ri,))
-            @show rowind_ri
           end
 
           Base.Cartesian.@nexprs $CUI ci -> begin
             colind = c+ci-1
             cx_ci = VectorizationBase.vbroadcast($W, VectorizationBase.vload(px, (colind,)))
             Base.Cartesian.@nexprs $RUI ri -> begin
               # rowind = MM($W, r + (ri-1)*$W)
-              m = colind < rowind_ri #(ro == co ? colind < rowind_ri : 0 < rowind_ri)
+              m = colind < rowind_ri
+              #m1 = (ro == co ? colind < rowind_ri : 0<rowind_ri)
```

Comment:
In general, until this `skewsymmetricmv.jl` file is working, I would remove it — just keep it on a branch / PR.  

This is called a [feature branch workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) — all code development should happen on branches, and only get merged into `main` when it is ready.

Reply:
I moved it to a dedicated branch.